did_document_format.proto

```diff
@@ -27,14 +27,14 @@ message Signature {
 message Service {
   string id = 1;
   string type = 2;
-  string serviceEndpoint = 3;
-  string data = 4;
+  optional string service_endpoint = 3;
+  optional string data = 4;
 }
 
 message Document {
   string id = 1;
   string controller = 2;
-  repeated VerificationMethod verificationMethods = 3;
+  repeated VerificationMethod verification_methods = 3;
```

Comment:
Why are you changing the field to snake-case format?
This is going to affect the final schema output of the did document some of the other languages.

Please revert to previous camelCase format.